evol/individual.py

```diff
@@ -40,14 +40,20 @@ def from_dict(cls, data: dict) -> 'Individual':
         result.id = data['id']
         return result
 
-    def evaluate(self, eval_function: Callable[..., float], lazy: bool=False):
+    def post_evaluate(self, result):
```

Comment:
Can we find a way to do this without adding a method to `Individual`? If not, we should make this method private (e.g. `_post_evaluate`).